src/main/java/dev/emi/trinkets/mixin/datafixer/Schema1460Mixin.java

```diff
@@ -0,0 +1,34 @@
+package dev.emi.trinkets.mixin.datafixer;
+
+import com.llamalad7.mixinextras.injector.ModifyReturnValue;
+import com.mojang.datafixers.DSL;
+import com.mojang.datafixers.schemas.Schema;
+import com.mojang.datafixers.types.templates.TypeTemplate;
+import net.minecraft.datafixer.TypeReferences;
+import net.minecraft.datafixer.schema.Schema1460;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Unique;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+import java.util.Map;
+import java.util.function.Supplier;
+
+@Mixin(Schema1460.class)
+public class Schema1460Mixin {
```

Comment:
I said it in discord but I do not care for this, I can't easily understand this part of the codebase, all of the methods are unmapped, and the benefit is pretty low level that I've never supported before, I'd prefer this class be removed.

Reply:
I, as a developer/user of this library prefer for Trinkets to support this, as losing things like item data/enchantments isn't fun. In cases where people use trinkets purely for things as elytra cape slot will likely want to not lose the data.

The only thing happening here is adding nbt structure of trinkets data to definitions of player and entity. It doesn't do anything else, as vanilla handles the rest. These methods are unnamed, as they are just lambdas, which never have a name in yarn.